jsk_unitree_robot/cross/build_ros1_dependencies.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+IMAGE_NAME="${IMAGE_NAME:-ros1-unitree}"
```

Comment:
This line is also necessary for build_ros1.sh

Reply:
Thanks. We have updated several patches to fix this issue. In the latest version (https://github.com/jsk-ros-pkg/jsk_robot/pull/1868/commits/d832815eb42af8e4518cf7116179104c0c2c578f), catkin_virtualenv 0.9.0 was installed in the System directory along with python3-rospkg-modules. Just now, I am checking to see if https://github.com/jsk-ros-pkg/jsk_robot/issues/1871, has solved the problem.

Reply:
Thanks you, I am also checking the building and am going to test build on MacOS